sentinel-core/src/test/java/com/alibaba/csp/sentinel/eagleeye/EagleEyeCoreUtilsTest.java

```diff
@@ -0,0 +1,164 @@
+        Assert.assertFalse(EagleEyeCoreUtils.isNotEmpty(null));
+
+        Assert.assertTrue(EagleEyeCoreUtils.isNotEmpty(" "));
+        Assert.assertTrue(EagleEyeCoreUtils.isNotEmpty("foo"));
+    }
+
+    @Test
+    public void testTrim() {
+        Assert.assertNull(EagleEyeCoreUtils.trim(null));
+
+        Assert.assertEquals("", EagleEyeCoreUtils.trim(""));
+        Assert.assertEquals("", EagleEyeCoreUtils.trim(" "));
+        Assert.assertEquals("foo", EagleEyeCoreUtils.trim("foo"));
+        Assert.assertEquals("foo", EagleEyeCoreUtils.trim("foo "));
+        Assert.assertEquals("foo", EagleEyeCoreUtils.trim(" foo"));
+        Assert.assertEquals("foo", EagleEyeCoreUtils.trim(" foo "));
+        Assert.assertEquals("f o o", EagleEyeCoreUtils.trim(" f o o "));
+    }
+
+    @Test
+    public void testSplit() {
+        Assert.assertNull(EagleEyeCoreUtils.split(null, 'a'));
+
+        Assert.assertArrayEquals(new String[0],
+                EagleEyeCoreUtils.split("", ','));
+        Assert.assertArrayEquals(new String[]{"foo", "bar", "baz"},
+                EagleEyeCoreUtils.split("foo,bar,baz", ','));
+    }
+
+    @Test
+    public void testAppendWithBlankCheck() {
+        Assert.assertEquals(new StringBuilder("bar").toString(),
+                EagleEyeCoreUtils.appendWithBlankCheck(
+                        null, "bar", new StringBuilder()).toString());
+        Assert.assertEquals(new StringBuilder("foo").toString(),
+                EagleEyeCoreUtils.appendWithBlankCheck(
+                        "foo", "bar", new StringBuilder()).toString());
+    }
+
+    @Test
+    public void testAppendWithNullCheck() {
+        Assert.assertEquals(new StringBuilder("bar").toString(),
+                EagleEyeCoreUtils.appendWithNullCheck(
+                        null, "bar", new StringBuilder()).toString());
+        Assert.assertEquals(new StringBuilder("foo").toString(),
+                EagleEyeCoreUtils.appendWithNullCheck(
+                        "foo", "bar", new StringBuilder()).toString());
+    }
+
+    @Test
+    public void testAppendLog() {
+        Assert.assertEquals(new StringBuilder("foo bar baz foo").toString(),
+                EagleEyeCoreUtils.appendLog(
+                        "foo\nbar\rbaz,foo", new StringBuilder(), ',')
+                        .toString());
+        Assert.assertEquals(new StringBuilder().toString(),
+                EagleEyeCoreUtils.appendLog(null, new StringBuilder(), ',')
+                        .toString());
+    }
+
+    @Test
+    public void testFormatTime() {
+        Assert.assertEquals("2019-06-15 20:13:14.000",
+                EagleEyeCoreUtils.formatTime(1560600794000L));
+    }
+
+    @Test
+    public void testGetSystemProperty() {
+        Assert.assertNull(EagleEyeCoreUtils.getSystemProperty(null));
+        Assert.assertNull(EagleEyeCoreUtils.getSystemProperty("foo"));
+    }
+
+    @Test
+    public void testGetSystemPropertyForLong() {
```

Comment:
Both asserts expect the `defaultValue`. Does this test the whole method?

Reply:
Oops.. Done now